src/conrollers/expenseController.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const userServices = require('../servises/userServices');
+const expenseServices = require('../servises/expenseServices');
+
+function getExpenses(req, res) {
+  const { userId, categories, from, to } = req.query;
+  const expenses = expenseServices.getExpenses(userId, categories, from, to);
+
+  res.send(expenses);
+};
+
+function getExpenseById(req, res) {
+  const foundExpense = expenseServices.getExpenseById(req.params.expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+function createExpense(req, res) {
```

Comment:
Use arrow functions instead of function declarations for consistency and shorter syntax.